core/src/main/java/com/interrupt/dungeoneer/game/Progression.java

```diff
@@ -15,8 +13,6 @@ public class Progression {
 	public int messagesFound = 0;
 	public boolean won = false;
 
-	public int goldAtStartOfRun = 0;
```

Comment:
Removing this causes no harm. We only get a warning: `[Serialization] Warning: Unknown field: goldAtStartOfRun for class: class com.interrupt.dungeoneer.game.Progression`

Reply:
Thanks for checking!